app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -2,6 +2,12 @@ module TwoFactorAuthenticatable
   extend ActiveSupport::Concern
   include TwoFactorAuthenticatableMethods
 
+  NEED_AUTHENTICATION = 'need_two_factor_authentication'.freeze
+  OTP_LENGTH = 6
+  ALLOWED_OTP_DRIFT_SECONDS = 30
+  DIRECT_OTP_VALID_FOR_SECONDS = AppConfig.env.otp_valid_for.to_i.minutes
+  REMEMBER_TFA_COOKIE = 'remember_tfa'.freeze
```

Comment:
We use `2fa` more than `tfa` in our code, WDYT of renaming?

```suggestion
  REMEMBER_2FA_COOKIE = 'remember_2fa'.freeze
```

Reply:
Changed the constant name, but I think the cookie key at least has to remain the same to avoid breaking existing cookies?

Reply:
oh yeah it would definitely need to remain the same -- I forgot the value of the cookie came from the gem